namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-template-typescript/resources/main.tf

```diff
@@ -5,15 +5,35 @@ terraform {
 
 provider "aws" {
   region = "eu-west-2"
+
+  default_tags {
+    tags = {
+      # see https://user-guide.cloud-platform.service.justice.gov.uk/documentation/getting-started/accessing-the-cloud-console.html
```

Comment:
Adding this comment because this is a template namespace for HMPPS - teams may use it for reference and they may not know what the param is used for